gno.land/cmd/gnoweb/pages/EVENTS.md

```diff
@@ -0,0 +1,114 @@
+# Events
+
+If you’re interested in building web3 with us, catch up with Gno.land in person at one of our industry events. We’re looking to connect with developers and like-minded thinkers who can contribute to the growth of our platform.
+
+---
+
+## Upcoming Events
+
+<div class="columns-3">
+
+<div>
+
+### Eth Seoul
+
+- **The Evolution of Smart Contracts: A Journey into Gno.land**
+- Seoul, June 3, 2023
+- Manfred Touron
+
+[Learn more](https://2023.ethseoul.org/)
+
+</div>
+
+<div>
+
+### BUIDL Asia
+
+- **Proof of Contribution in Gno.land**
+- Seoul, June 6, 2023
+- Manfred Touron
+
+[Learn more](https://www.buidl.asia/)
+
+</div>
+
+<div>
+
+### EthCC
+
+- **Come Meet Us at our Booth**
+- Paris, July 17 - 20, 2023
+- Manfred Touron
+
+[Learn more](https://www.ethcc.io/)
+
+</div>
+</div>
+
+---
+
+## Past Events
+
+<div class="columns-3">
+
+<div>
+
+### Game Developer Conference
+
+- **Side Event: Web3 Gaming Apps Powered by Gno**
+- San Francisco, Mach 23, 2023
+- Jae Kwon
+
+[Watch the talk](https://www.youtube.com/watch?v=IJ0xel8lr4c)
+
+</div>
+
+<div>
+
+### EthDenver
+
+- **Side Event: Discover Gno.land**
+- Denver, Feb 24 - Mar 5, 2023
+- Jae Kwon
+
+[Watch the talk](https://www.youtube.com/watch?v=IJ0xel8lr4c)
+
+</div>
+<div>
```

Comment:
even if it's more code and potentially useless, I think we should encourage naming divs or adding comments.

```suggestion
</div>
<div class="column">
```

or

```suggestion
</div>
<div> <!-- column -->
```

Reply:
That's a good idea to improve readability! Let's do it.